src/System.Globalization/tests/DateTimeFormatInfo/DateTimeFormatInfoGetAbbreviatedDayName.cs

```diff
@@ -16,8 +16,7 @@ public static IEnumerable<object[]> GetAbbreviatedDayName_TestData()
             yield return new object[] { new CultureInfo("en-US").DateTimeFormat, englishAbbreviatedDayNames };
             yield return new object[] { new DateTimeFormatInfo(), englishAbbreviatedDayNames };
 
-            // ActiveIssue(2103)
-            if (!PlatformDetection.IsUbuntu1510 && !PlatformDetection.IsUbuntu1604 && !PlatformDetection.IsUbuntu1610)
+            if (!PlatformDetection.IsUbuntu || PlatformDetection.IsUbuntu1404)
```

Comment:
Nit: At this point I'm not sure we need to care as much about specialized test cases for 14.04 (e.g. this could just be `if (!PlatformDetection.IsUbuntu)`, but I'm ok leaving it like this for now.